app/src/main/java/no/usn/mob3000/ui/screens/SettingsScreen.kt

```diff
@@ -127,7 +127,8 @@ fun SettingsScreen(
 
             Button(
                 onClick = onAdminDashboardClick,
-                colors = ButtonDefaults.buttonColors(DefaultButton),
+                colors = ButtonDefaults.buttonColors(
+                    containerColor = MaterialTheme.colorScheme.primary),
```

Comment:
Remember indentation. Specifically, the closing parenthesis should always be on a newline in this sort of use-case.